src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -144,7 +144,8 @@ class MultipleFields extends Component {
    */
   addAnotherItem = () => {
     const { value, onChange } = this.props;
-    const newValue = [...value, ''];
+    const updatedValue = (value.length && value) || [''];
```

Comment:
I am confused, doesn't this result in newValue being potentially``` ['', '']```?

Reply:
Hi @dawehner 
I apologise for the late reply. You are right, it will return `['', '']`  but only if the redux state is a step behind the local state of MultipleFields. It works fine once both are synched.

Synching the local state with the redux breaks the restore functionality (also mentioned by Vijay).